### ISSUE TITLE
Run crc on one machine the use oc against it from another 

### ISSUE THREAD

**Issue by jeffsaremi** [code-ready/crc#1071]
### General information

  * OS: Windows
  * Hypervisor: Hyper-V 
  * Did you run `crc setup` before starting it (Yes/No)? Yes

## CRC version
```bash
crc version: 1.6.0+8ef676f
OpenShift version: 4.3.0 (embedded in binary)
```
  
## CRC status
```bash
C:\crc-windows-1.6.0-amd64>crc status
CRC VM:          Running
OpenShift:       Running (v4.3.0)
Disk Usage:      9.628GB of 32.72GB (Inside the CRC VM)
Cache Usage:     12.69GB
Cache Directory: C:\Users\jesaremi\.crc\cache
```

## CRC config
```bash
C:\crc-windows-1.6.0-amd64>crc config view

```

## Host Operating System
```bash
C:\crc-windows-1.6.0-amd64>systeminfo

Host Name:                 JESAREMI-HP
OS Name:                   Microsoft Windows 10 Enterprise
OS Version:                10.0.18363 N/A Build 18363
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Member Workstation
OS Build Type:             Multiprocessor Free
Registered Owner:          Microsoft Corp.
Registered Organization:   Microsoft IT
Product ID:                00330-80000-00000-AA800
Original Install Date:     7/30/2019, 11:45:53 AM
System Boot Time:          2/15/2020, 8:06:58 PM
System Manufacturer:       HP
System Model:              HP Z2 Tower G4 Workstation
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: Intel64 Family 6 Model 158 Stepping 10 GenuineIntel ~3192 Mhz
BIOS Version:              HP Q50 Ver. 01.05.04, 6/17/2019
Windows Directory:         C:\windows
System Directory:          C:\windows\system32
Boot Device:               \Device\HarddiskVolume4
System Locale:             en-us;English (United States)
Input Locale:              en-us;English (United States)
Time Zone:                 (UTC-08:00) Pacific Time (US & Canada)
Total Physical Memory:     65,387 MB
Available Physical Memory: 28,024 MB
Virtual Memory: Max Size:  75,115 MB
Virtual Memory: Available: 33,935 MB
Virtual Memory: In Use:    41,180 MB
Page File Location(s):     C:\pagefile.sys
Domain:                    redmond.corp.microsoft.com
Logon Server:              \\CY1-RED-DC-03
Hotfix(s):                 16 Hotfix(s) Installed.
                           [01]: KB4534132
                           [02]: KB4493478
                           [03]: KB4497727
                           [04]: KB4503308
                           [05]: KB4508433
                           [06]: KB4509096
                           [07]: KB4515383
                           [08]: KB4516115
                           [09]: KB4517245
                           [10]: KB4520390
                           [11]: KB4521863
                           [12]: KB4524569
                           [13]: KB4528759
                           [14]: KB4537759
                           [15]: KB4538674
                           [16]: KB4532693
Network Card(s):           4 NIC(s) Installed.
                           [01]: Intel(R) Ethernet Connection (7) I219-LM
                                 Connection Name: Ethernet
                                 DHCP Enabled:    Yes
                                 DHCP Server:     N/A
                                 IP address(es)
                           [02]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (Default Switch)
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 192.168.207.193
                                 [02]: fe80::811c:dc54:cd74:642e
                           [03]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (external-1)
                                 DHCP Enabled:    Yes
                                 DHCP Server:     255.255.255.255
                                 IP address(es)
                                 [01]: 169.254.119.54
                                 [02]: fe80::a4dc:b7c0:b6a4:7736
                           [04]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (crc)
                                 DHCP Enabled:    Yes
                                 DHCP Server:     10.221.228.18
                                 IP address(es)
                                 [01]: 10.91.90.45
                                 [02]: fe80::1955:537c:a2dd:d076
                                 [03]: 2001:4898:e0:bf:e4f8:ba04:a787:53fb
                                 [04]: 2001:4898:e0:bf:1955:537c:a2dd:d076
Hyper-V Requirements:      A hypervisor has been detected. Features required for Hyper-V will not be displayed.

```

### Steps to reproduce

 Here's the problem we have:
- based on what we were told we cannot install crc on ubuntu 16.04 because libvirt is old.
- so we're trying the following configuration where the crc is installed on the developer's desktop (Windows) and on ubuntu the person can use oc client (which runs on ubuntu 16.04) to connect and do what is needed.
- For that we copy kubeconfig from C:\users\user_name\.crc\cache\crc_hyperv_4.3.0 to the ubuntu machine.
- setup dns entries using the following:

$ cat /etc/NetworkManager/conf.d/crc-nm-dnsmasq.conf
[main]
dns=dnsmasq
$ cat /etc/NetworkManager/dnsmasq.d/crc.conf
server=/apps-crc.testing/10.91.90.161
server=/crc.testing/10.91.90.161

and then try connecting. We get this:

```
$ oc login -u kubeadmin -p '...'
error: dial tcp: lookup api.crc.testing on 10.193.8.10:53: no such host - verify you have provided the correct host and port and that the server is currently running.
```

### Expected
oc login succeeds.

### Actual
Fails on networking

### Logs

You can start crc with `crc start --log-level debug` to collect logs.
Please consider posting this on http://gist.github.com/ and post the link in the issue.

**Comment by praveenkumar** [comment]
This is duplicate of #705 , closing this. @jeffsaremi can you please add you request on that bug. Also as per your bug description looks like you just want to expose CRC to use `oc` command from Ubuntu, `oc` command is cross compiled and available for windows also.
